test/aleph/netty_test.clj

```diff
@@ -0,0 +1,14 @@
+(ns aleph.netty-test
+  (:require
+   [aleph.netty :as netty]
+   [clojure.test :refer [deftest is]]
+   [manifold.stream :as s])
+  (:import
+   (io.netty.channel.embedded EmbeddedChannel)))
+
+(deftest closing-a-channel-sink
+  (let [ch (EmbeddedChannel.)
+        s (netty/sink ch)]
+    (is (= true @(s/put! s "foo")))
+    (is (nil? @(netty/wrap-future (netty/close ch))))
+    (is (= false @(s/put! s "foo")))))
```

Comment:
While this is fine as is, I'll still bet that if you fire up two threads, have them race, and repeat a few thousand times, you'll see an error.

Reply:
FTR: This was based on a slight misunderstanding on my end, see https://github.com/clj-commons/aleph/issues/618#issuecomment-1224206493

I suggest to leave the test case in there anyway. Could be a nice blueprint for further tests around the `aleph.netty` namespace.